homeassistant/components/renson/entity.py

```diff
@@ -0,0 +1,42 @@
+"""Entity class for Renson ventilation unit."""
+from __future__ import annotations
+
+from renson_endura_delta.field_enum import (
+    DEVICE_NAME_FIELD,
+    FIRMWARE_VERSION_FIELD,
+    HARDWARE_VERSION_FIELD,
+)
+from renson_endura_delta.renson import RensonVentilation
+
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import RensonCoordinator
+from .const import DOMAIN
+
+
+class RensonEntity(CoordinatorEntity):
+    """Renson entity."""
+
+    def __init__(
+        self, name: str, api: RensonVentilation, coordinator: RensonCoordinator
+    ) -> None:
+        """Initialize the Renson entity."""
+        super().__init__(coordinator)
+
+        self._attr_device_info = DeviceInfo(
+            identifiers={(DOMAIN, "sensor")},
```

Comment:
The identifier need to have a unique and stable string as the second item in the tuple.

`"sensor"` is not unique for this device.

A common device identifier is a serial number.

https://developers.home-assistant.io/docs/device_registry_index#device-properties